### PR TITLE
PMM-10381 Fixed postgres initialization inside container

### PR DESCRIPTION
**Pull request by gen1us2k**
PMM-10381

Build: SUBMODULES-2673

- [ ] Links to other linked pull requests (optional).

### CODE REVIEW

**Comment by nikita-b** [comment]
@gen1us2k We'll break setup.py for `dev-latest` with this fix:
```
[root@3763b871a5c5 opt]# ls /srv/ | grep postgres
postgres14
```
What do you want to fix?

**Comment by gen1us2k** [comment]
>What do you want to fix?
So. dev-latest doesn't work in minikube and helm-chart tests are failing without those changes. I found inconsistencies between entrypoint and setup.py files.

`setup.py` points to /srv/postgres14 directory, however, entrypoint initializes the database in /srv/postgres directory which leads to failing postgres with this error

```
postgres: could not access directory "/srv/postgres14": No such file or directory 
```

My first attempt was to fix it in entrypoint and this is the second try

File: .devcontainer/setup.py [hunk 1]
```diff
@@ -90,8 +90,8 @@ def setup():
     """Runs various setup commands."""
     run_commands([
         # allow connecting from any host, needed to connect from host to PG running in docker
-        "sed -i -e \"s/#listen_addresses = \'localhost\'/listen_addresses = \'*\'/\" /srv/postgres14/postgresql.conf",
-        "echo 'host    all         all     0.0.0.0/0     trust' >> /srv/postgres14/pg_hba.conf",
+        "sed -i -e \"s/#listen_addresses = \'localhost\'/listen_addresses = \'*\'/\" /srv/postgres/postgresql.conf",
+        "echo 'host    all         all     0.0.0.0/0     trust' >> /srv/postgres/pg_hba.conf",
```
Comment (idoqo) [inline .devcontainer/setup.py:94]: Not sure why we are reverting it here. Postgres in devcontainer was upgraded recently (https://github.com/percona/pmm/pull/1051). 

Also, running `SHOW config_file` here gives this:

```
[root@pmm-managed-server pmm]# psql -U postgres -c 'SHOW config_file'
           config_file
---------------------------------
 /srv/postgres14/postgresql.conf
(1 row)

[root@pmm-managed-server pmm]#
```

Comment (idoqo) [reply .devcontainer/setup.py:94]: I see that Nikita raised similar concern above already.

**Comment by gen1us2k** [comment]
@nikita-b As another option, I can fix it in entrypoints. What do you suggest here?

**Comment by nikita-b** [comment]
@gen1us2k yep, you are right. It's issue with entrypoint. Can you fix it there?

**Comment by gen1us2k** [comment]
@nikita-b Fixed it.

**Review by nikita-b — APPROVED** [review]

**Review by idoqo — APPROVED** [review]

**Pull request merged**